<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsageStatistic::Internal::UsageStatisticWidget</class>
 <widget class="QWidget" name="UsageStatistic::Internal::UsageStatisticWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="gbTelemetryMode">
     <property name="title">
      <string>&amp;Telemetry mode</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="cbMode">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteTelemetryLvlDescription">
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDataSources">
     <property name="title">
      <string>Data &amp;sources</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
      <item>
       <widget class="QListWidget" name="lvDataSources"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lblDataSourceDescription">
            <property name="text">
             <string>&amp;Description</string>
            </property>
            <property name="buddy">
             <cstring>pteDataSourceDescription</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="pteDataSourceDescription">
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lblCollectedData">
            <property name="text">
             <string>&amp;Collected Data</string>
            </property>
            <property name="buddy">
             <cstring>pteCollectedData</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="pteCollectedData">
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblPrivacyPolicy">
       <property name="text">
        <string>Legal Notice and Privacy Policy</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
